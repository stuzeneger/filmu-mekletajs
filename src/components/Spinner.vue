<template>
  <div class="spinner"></div>
</template>

<script lang="ts">
export default {}
</script>

<style lang="scss" scoped>
@import "@/assets/styles/spinner.scss";
</style>
