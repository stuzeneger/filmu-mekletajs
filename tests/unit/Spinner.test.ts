import { mount } from '@vue/test-utils'
import { describe, it, expect } from 'vitest'
import Spinner from '@/components/Spinner.vue'

describe('Spinner.vue', () => {
    it('renders the spinner element', () => {
        const wrapper = mount(Spinner)
        expect(wrapper.classes()).toContain('spinner')
    })

    it('matches snapshot', () => {
        const wrapper = mount(Spinner)
        expect(wrapper.html()).toMatchSnapshot()
    })
})
