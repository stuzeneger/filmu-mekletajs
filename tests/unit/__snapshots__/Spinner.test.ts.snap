// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Spinner.vue > matches snapshot 1`] = `"<div data-v-259be2b2=\\"\\" class=\\"spinner\\"></div>"`;
